File: typescript/src/api/resources/evaluationResults/types/ListResultsEvaluationResultsRequestEnvironment.ts

export const ListResultsEvaluationResultsRequestEnvironment = {
        Production: "production",
        Staging: "staging",
        Development: "development"
    } as const;
export type ListResultsEvaluationResultsRequestEnvironment = typeof ListResultsEvaluationResultsRequestEnvironment[keyof typeof ListResultsEvaluationResultsRequestEnvironment];

File: ruby/lib/Sikaru/evaluation_results/types/list_results_evaluation_results_request_environment.rb
# frozen_string_literal: true

module Sikaru
  module EvaluationResults
    module Types
      module ListResultsEvaluationResultsRequestEnvironment
        extend Sikaru::Internal::Types::Enum

        PRODUCTION = "production"
        STAGING = "staging"
        DEVELOPMENT = "development"
      end
    end
  end
end
